app/src/main/java/org/rudirect/android/util/RUDirectUtil.java

```diff
@@ -44,4 +46,20 @@ public static int dpToPx(int dp) {
         DisplayMetrics displayMetrics = RUDirectApplication.getContext().getResources().getDisplayMetrics();
         return (int) ((dp * displayMetrics.density) + 0.5);
     }
+
+    // Gets stop nearest to location.
+    public static BusStop getNearestStop(Location location) {
+        double minDist = Double.MAX_VALUE;
+        BusStop closestStop = null;
+        for (BusStop stop : RUDirectApplication.getBusData().getBusStops()) {
+            double dist = Math.sqrt(Math.pow(Double.parseDouble(stop.getLatitude()) - location.getLatitude(), 2) +
```

Comment:
Hm, not sure: for OpenJDK at least, they have [special handling](http://hg.openjdk.java.net/jdk8u/jdk8u/hotspot/file/5755b2aee8e8/src/share/vm/opto/library_call.cpp#l1799) for the `y == 2` case that just inlines `x * x`.

I'll switch it out anyway, and kill the `sqrt` while I'm at it.